client/image_save.go

```diff
@@ -9,9 +9,15 @@ import (
 
 // ImageSave retrieves one or more images from the docker host as an io.ReadCloser.
 // It's up to the caller to store the images and close the stream.
-func (cli *Client) ImageSave(ctx context.Context, imageIDs []string) (io.ReadCloser, error) {
+func (cli *Client) ImageSave(ctx context.Context, names []string) (io.ReadCloser, error) {
+	for _, name := range names {
+		if _, err := parseNamed(name); err != nil {
+			return nil, err
```

Comment:
Same here; I think I'd prefer to keep this validation on the daemon; is there a specific case that's not caught?